packages/testing/src/api/index.ts

```diff
@@ -49,10 +56,11 @@ export function initializeAdminApp(options: any): admin.app.App {
   );
 }
 
-export function initializeTestApp(options: any): admin.app.App {
-  if (!('databaseName' in options)) {
-    throw new Error('databaseName not specified');
-  }
+export type TestAppOptions = {
```

Comment:
I would suggest to make this an interface or to just inline it and use two arguments in `initializeTestApp`.

Reply:
Almost :) But it looks fine the way it is.